### PR TITLE
Blink esp8266

### PR DESCRIPTION
**Pull request by adithya-s-k**
## Related Issue 

- Info about the related issue 

- [x] SSOC Participant  
- [ ] Contributor

Closes: #291 that will be closed through this PR

### Describe the changes you've made

I have added blinking led for ESP8266

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

I have tested it on my ESP8266(Hardware)

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by mdnazisharman2803 — APPROVED** [review]

**Comment by adithya-s-k** [comment]
@prathimacode-hub the pull request has been approved can you please merge it.

File: ESP8266/README.md [hunk 1]
```diff
@@ -1 +1,45 @@
 
+# Welcome to IoT-Spot👋
```
Comment (Harshil-Jani) [inline ESP8266/README.md:2]: Delete this README from the `ESP8266/README.md` This got copied from the main README of the project

**Review by Harshil-Jani — CHANGES_REQUESTED** [review]

File: ESP8266/README.md (hunk 1 above)
Comment (adithya-s-k) [reply ESP8266/README.md:2]: I followed the main readme file but i have made the necessary changes for the ESP8266
should I delete the whole file? 
@Harshil-Jani

**Review by mdnazisharman2803 — APPROVED** [review]

**Comment by adithya-s-k** [comment]
@Harshil-Jani please look into the changes

**Review by Harshil-Jani — APPROVED** [review]

**Pull request merged**